src/Runner.Worker/GitHubContext.cs

```diff
@@ -22,8 +22,8 @@ public sealed class GitHubContext : DictionaryContextData, IEnvironmentContextDa
             "repository_owner",
             "run_id",
             "run_number",
+            "server_url",
             "sha",
-            "url",
```

Comment:
can we safely remove GITHUB_URL?

Reply:
what about checkout action?

Reply:
or anything already using GITHUB_URL

Reply:
I added back compat code to the [checkout pr](https://github.com/actions/checkout/pull/248) (fallback order: `$GITHUB_SERVER_URL || $GITHUB_URL || 'https://github.com'`

GHES Alpha is pinned to the 2.169.0 runner, but uses checkout from dotcom. So only checkout needs to hold compat (can delete after alpha support is over)

Nothing else uses GITHUB_URL

Reply:
Feature flag only made it out to Ring 1